test/server/cards/03_TWI/events/NowThereAreTwoOfThem.spec.ts

```diff
@@ -0,0 +1,69 @@
+describe('Now There are Two of Them', function() {
+    integration(function(contextRef) {
+        describe('Now There are Two of Them\'s ability -', function() {
+            it('should allow you to play a non-Vehicle unit from your hand that shares a Trait with the unit you control for 5 less, if you have only one unit in play', function() {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['now-there-are-two-of-them', 'zuckuss#bounty-hunter-for-hire', 'cad-bane#hostage-taker', 'plo-koon#kohtoyah', 'fetts-firespray#pursuing-the-bounty'],
```

Comment:
Could we put in at least one card that is just Bounty Hunter and just Underworld so we can confirm that the trait matching logic works in all cases? Also we should populate the opponent's hand with at least one valid target to make sure it's not getting included accidentally